themes-default/slim/views/addShows_newShow.mako

```diff
@@ -370,9 +370,8 @@ const startVue = () => {
                         alreadyAdded = (() => {
                             if (!alreadyAdded) return false;
                             // Extract existing show info
-                            const [ mIndexerId, mShowId ] = alreadyAdded;
-                            const indexerIdentifier = indexers[mIndexerId] ? indexers[mIndexerId].identifier : mIndexerId;
-                            return 'home/displayShow?indexername=' + indexerIdentifier + '&seriesid=' + mShowId;
+                            const [ mIndexerName, mShowId ] = alreadyAdded;
```

Comment:
What's up with the `m` at the start of the var name?

Reply:
I think it was `m` for matched.
I just wanted to distinguish those vars from the ones up top.
I realize it's not really needed, but I feel like it's a bit more readable like that.

Reply:
Since we're in a different scope maybe change them over or use `match` since it's really that obvious the `m` is for?